libs/components/ag-grid/src/lib/modules/ag-grid/header/header.component.ts

```diff
@@ -30,28 +30,54 @@ import { SkyAgGridHeaderParams } from '../types/header-params';
   selector: 'sky-ag-grid-header',
   templateUrl: './header.component.html',
   styleUrls: ['./header.component.scss'],
-  changeDetection: ChangeDetectionStrategy.OnPush,
+  host: {
+    '[attr.title]': 'accessibleHeaderText()',
+    '[attr.aria-label]': 'displayName() || accessibleHeaderText()',
+    '[attr.role]': '"note"',
+  },
```

Comment:
I started down the path thinking it was a change detection issue in their test, so I switched to signals and removed `OnPush` b/c AG Grid renderers don't use Angular inputs.

The issue ended up being `role="presentation"` being added by AG Grid negates the content of this component as counting towards readable text for the column heading. Using `aria-label` counters that effect.

It's using `role="note"` because a role is required for `aria-label`, the [note role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/note_role) seemed innocuous enough, and it fits the need.